### PR TITLE
RR-780 - In Prison Work Interests page

### PR DESCRIPTION
**Pull request by nathanrussell-moj-digital**
This PR implements the In Prison Work interests page for the short question set

The only part that is missing is the update to the short question set cypress test, because at the moment we cannot get to this page (we are creating these last few pages out of sequence, but I'll come back round at the end and write the cypress test step 👍 )

### CODE REVIEW

File: server/routes/induction/create/inPrisonWorkCreateController.test.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      await controller.getInPrisonWorkView(req, res, next)
+
+      // Then
+      expect(res.render).toHaveBeenCalledWith('pages/induction/inPrisonWork/index', expectedView)
+      expect(req.session.inPrisonWorkForm).toBeUndefined()
+      expect(req.session.inductionDto).toEqual(inductionDto)
+    })
+  })
+
+  describe('submitInPrisonWorkForm', () => {
```
Comment (leejacobson-moj) [inline server/routes/induction/create/inPrisonWorkCreateController.test.ts:96]: Does this need a unit test for a successful form submission, or are you going to pick that up in another PR?

**Review by leejacobson-moj — APPROVED** [review]
Looks good. Approved 👍

File: server/routes/induction/create/inPrisonWorkCreateController.test.ts (hunk 1 above)
Comment (nathanrussell-moj-digital) [reply server/routes/induction/create/inPrisonWorkCreateController.test.ts:96]: Doh! Did I not do that? 
Good spot! I'll do that now 👍

Comment (nathanrussell-moj-digital) [reply server/routes/induction/create/inPrisonWorkCreateController.test.ts:98]: Done

**Review by leejacobson-moj — APPROVED** [review]
Approved

**Pull request merged**